turbo/jsonrpc/eth_receipts.go

```diff
@@ -362,21 +362,13 @@ func (api *BaseAPI) getLogsV3(ctx context.Context, tx kv.TemporalTx, begin, end
 		rawLogs := exec.GetRawLogs(txIndex)
 
 		// `ReadReceipt` does fill `rawLogs` calulated fields. but we don't need it anymore.
-		r, err := rawtemporaldb.ReceiptAsOfWithApply(tx, txNum, rawLogs, txIndex, blockHash, blockNum, txn)
+		filledLogs, err := rawtemporaldb.PopulateLogsIndices(tx, txNum, rawLogs, txIndex, blockHash, blockNum, txn)
```

Comment:
i would advise to add nil-check of `filledLogs` variable - so in future we will not get panic.